### PR TITLE
Update contributors table

### PR DESCRIPTION
**Pull request by AlexeyVatolin**
## Checklist

- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 

### Adding datasets checklist

**Reason for dataset addition**: ... 

- [ ] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb -m {model_name} -t {task_name}` command.
  - [ ] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [ ] `intfloat/multilingual-e5-small`
- [ ] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [ ] If the dataset is too big (e.g. >2048 examples), considering using `self.stratified_subsampling() under dataset_transform()`
- [ ] I have filled out the metadata object in the dataset file (find documentation on it [here](https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md#2-creating-the-metadata-object)).
- [ ] Run tests locally to make sure nothing is broken using `make test`. 
- [ ] Run the formatter to format the code using `make lint`. 

### Adding a model checklist

 - [ ] I have filled out the ModelMeta object to the extent possible
 - [ ] I have ensured that my model can be loaded using
   - [ ] `mteb.get_model(model_name, revision_id)` and
   - [ ] `mteb.get_model_meta(model_name, revision_id)`
 - [ ] I have tested the implementation works on a representative set of tasks.

### CODE REVIEW

File: mteb/tasks/Classification/zho/CMTEBClassification.py
Comment (KennethEnevoldsen) [inline mteb/tasks/Classification/zho/CMTEBClassification.py:1]: I merged this in #1190 based on your comment in the issue before I saw this PR.

File: mteb/tasks/Classification/eng/LegalBenchClassification.py [hunk 1]
```diff
@@ -3026,152 +3026,6 @@ def dataset_transform(self):
         self.dataset = self.dataset.rename_column("answer", "label")
 
 
-class Diversity1LegalBenchClassification(AbsTaskClassification):
```
Comment (KennethEnevoldsen) [inline mteb/tasks/Classification/eng/LegalBenchClassification.py:3029]: I don't think we should remove these. Though I agree that they are very challenging tasks (and potentially problematically so).

**Review by KennethEnevoldsen — CHANGES_REQUESTED** [review]

File: mteb/tasks/Classification/zho/CMTEBClassification.py
Comment (AlexeyVatolin) [reply mteb/tasks/Classification/zho/CMTEBClassification.py:1]: I forgot to link this pull request to issue. I remove the changes and leave only the update of the Contributors table. Please update it in the article too

Comment (KennethEnevoldsen) [reply mteb/tasks/Classification/zho/CMTEBClassification.py:1]: Will update the article as well once we have the last contributors in (but have a script for it so if you are in the table and have more than 10 points you should get in)

**Review by KennethEnevoldsen — APPROVED** [review]

File: mteb/tasks/Classification/zho/CMTEBClassification.py
Comment (AlexeyVatolin) [reply mteb/tasks/Classification/zho/CMTEBClassification.py:1]: Yeah, I've got more than 10 points

Comment (KennethEnevoldsen) [reply mteb/tasks/Classification/zho/CMTEBClassification.py:1]: Just to make sure I reran the author list code in this [PR](https://github.com/embeddings-benchmark/mteb/pull/1194) and manually checked that your name appear on the author list.

**Pull request merged**